ChronoAPI/ChronoLog/include/chronolog_errcode.h

```diff
@@ -0,0 +1,31 @@
+//
+// Created by kfeng on 12/19/22.
+//
+
+#ifndef CHRONOLOG_CHRONOLOG_ERRCODE_H
+#define CHRONOLOG_CHRONOLOG_ERRCODE_H
+namespace chronolog
+{
+enum ErrorCode
+{
+    CL_SUCCESS = 0,                      // Success
```

Comment:
Please keep a similar alignment as the original errocode.h. I think that is more readable. If any line is longer than 120, pls adjust the spaces between error names and =.

Reply:
I do not understand this properly. Do you mean having all numbers in the same column?
The space between error names and = is the one established in the code style. I mean, the file is based on code style. Do I have something wrong on my format?

Reply:
I thought the style allows a aligned number column. but it does not. then pls keep the current version.